tests/unit/network.test.ts

```diff
@@ -0,0 +1,113 @@
+import { getL1Network, addCustomNetwork, getL2Network } from '../../src'
+
+import { expect } from 'chai'
+import { getNetwork } from '../../src/lib/dataEntities/networks'
+
+const mainnetId = 1
+const arbOneId = 42161
+const mockOrbitChainId = 99999999
+
+describe('Network', () => {
+  const fetchErrorMessage =
+    'Network fetched successfully but the chain ID is invalid.'
+
+  it('Adds a custom Orbit chain', async function () {
```

Comment:
nit: across the code base's test files, the `it` test cases starts with lower case